doc/source/ray-core/doc_code/neuron_core_accelerator.py

```diff
@@ -0,0 +1,28 @@
+# __neuron_core_accelerator_start__
+import ray
+import os
+from ray.util.accelerators import AWS_NEURON_CORE
+
+# On trn1.2xlarge instance, there will be 2 neuron cores.
+ray.init(resources={"num_neuron_cores": 2})
```

Comment:
In order to be consistent with the other resources in the `resources` dict, I would recommend using `"neuron_cores"` here instead of `"num_neuron_cores"`.

While it is true that we are using `ray.init(num_cpus=..., num_gpus=...)` to make it clearer that we are referring to a number, if you look in the documentation you will find that the convention for the resources dict is that we don't include `num_` for custom resources. Let's stick to that convention :)

cc @scv119

Reply:
should it be `neuron_cores` or `neuron_core`?

Reply:
Refactored to `neuron_cores`